src/architecture/consensus.md

```diff
@@ -59,9 +59,10 @@ The process of reaching consensus on the next block (at the blockchain height
 `H`) consists of several **rounds**, numbered from 1\. The first round starts
 once the
 validator commits the block at height `H - 1`. The onsets of rounds are
-determined
-by a fixed timetable: rounds start after regular intervals. As there is no
-global time, rounds may start at different time for different validators.
+determined by the following timetable: rounds start with time intervals
+that linearly increase with the round number.
+As there is no global time,
+rounds may start at different time for different validators.
```

Comment:
Props for using semantic line breaking. I'd like to see that more often with text files.